janus_core/calculations/md.py

```diff
@@ -23,11 +23,12 @@
 from janus_core.calculations.geom_opt import optimize
 from janus_core.helpers.janus_types import Ensembles, PathLike
 from janus_core.helpers.log import config_logger
+from janus_core.helpers.utils import FileNameMixin
```

Comment:
```suggestion
from janus_core.helpers.utils import FileNameMixIn
```

Reply:
"Mixin" is generally assumed to be one word in most programming contexts I've seen, hence the choice. 
https://en.wikipedia.org/wiki/Mixin

Happy to change, but not sure if wanted.